testsuite/features/step_definitions/command_steps.rb

```diff
@@ -403,15 +404,14 @@
   $channels_synchronized.each do |channel|
     log "I wait until '#{channel}' synchronized channel has finished"
     repeat_until_timeout(timeout: 7200, message: "Channel '#{channel}' not fully synced") do
-      # products.xml is the last file to be written when the server synchronize a channel,
-      # therefore we wait until it exist
```

Comment:
# solv is the last file to be written when the server synchronize a channel,
      # therefore we wait until it exist

Reply:
One of the last files maybe?
According to timestamps it could even be one of the earliest generated files.

Reply:
My source of info comes from here:
https://suse.slack.com/archives/C02DDMY6R0R/p1658829413385679